src/main/groovy/com/okta/test/mock/Config.groovy

```diff
@@ -24,6 +24,7 @@ class TestScenario {
     List<String> disabledTests = new ArrayList<>()
     Boolean enabled
     String command
+    String loginRedirectPath
     List<String> args = new ArrayList<>()
     Map<String, Integer> ports = new HashMap<>()
 }
```

Comment:
We are mixing having newlines and not having newlines at the end of files. Can we pick one?

Reply:
I was also confused about this. So, do we have or NOT have new lines at end of files?

Reply:
We have several JS lint rules that error if there are no newlines, so I'd advocate adding it. :)

+1 or -1 @bdemers or @robertjd?